params/config.go

```diff
@@ -442,6 +483,7 @@ func getChainConfig(networkID uint32, chainID *big.Int) *ChainConfig {
 		BanffBlockTimestamp:             getUpgradeTime(networkID, version.BanffTimes),
 		CortinaBlockTimestamp:           getUpgradeTime(networkID, version.CortinaTimes),
 		DurangoBlockTimestamp:           getUpgradeTime(networkID, version.DurangoTimes),
```

Comment:
nit: I am in favor of renaming all these "Timestamp" to "Time", it would make many lines shorter

Reply:
sure